lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -4029,7 +4033,8 @@ DefConstructor('\lx@mung@bibliography@pre', sub {
     if ($tag =~ /^ltx:(?:itemize|enumerate|description)$/) {
       $document->maybeCloseElement($tag); }    # Or even remove (if empty)?
     return; });
-DefConstructorI('\lx@bibnewblock', undef, "<ltx:bibblock>");
+DefConstructorI('\lx@bibnewblock', undef, sub {
+    $_[0]->openElement('ltx:bibblock') if $_[0]->isOpenable('ltx:bibblock'); });
```

Comment:
shouldn't we open _something_ when a block is not openable? Just to keep the contents grouped. Maybe a generic block or inline-block ?

Reply:
The scenario seems to be that we've somehow already gotten into a bibitem and block when this is needed.

Reply:
Oh, great then, nevermind 👍🏻